src/components/NotePopup/NotePopup.js

```diff
@@ -79,7 +79,7 @@ const NotePopup = ({ annotation, setIsEditing }) => {
   };
 
   const handleStateUpdate = state => {
-    const author = core.getCurrentUser();
+    const author = core.getDisplayAuthor(annotation);
```

Comment:
Hmm just to confirm with you that this is what we want because `getDisplayAuthor` returns the author of the annotation but `getCurrentUser` returns the current annotation(log in) user. They might not be the same.

Reply:
Ahh, good catch. How would one get the display author name of a current user?